src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -79,6 +79,9 @@ class ManagedInstancePreparer(AbstractPreparer, SingleValueReplacer):
     subscription_id = '4cac86b0-1e56-48c2-9df2-669a6d2d87c5'
     location = 'westeurope'
     subnet = '/subscriptions/4cac86b0-1e56-48c2-9df2-669a6d2d87c5/resourceGroups/Committer-SwaggerAndGeneratedSDKs-MI-CLI/providers/Microsoft.Network/virtualNetworks/vnet-powershell-cli-testing/subnets/ManagedInstance'
+    target_vnet_name = 'vnet-powershell-cli-testing'
+    target_subnet_name = 'ManagedInstance2'
+    target_subnet = '/subscriptions/4cac86b0-1e56-48c2-9df2-669a6d2d87c5/resourceGroups/Committer-SwaggerAndGeneratedSDKs-MI-CLI/providers/Microsoft.Network/virtualNetworks/{}/subnets/{}'.format(target_vnet_name, target_subnet_name)
```

Comment:
Can you use `az network vnet create` or `@VirtualNetworkPreparer` to create required subnet for tests? Using fixed subnet resource may crash when the resource is deleted by accident and it's hard for cli team to record tests since we don't have access to these resources.